js/app.js

```diff
@@ -64,9 +54,7 @@ Bartender.prototype.makeDrink = function() {
 Bartender.prototype.makeQuestions = function(questions) {
     //go through the questions and print them for the user
     for (item in Question) {
-        //for(i=0; i< Question.length; i++) {
-        console.log(item, Question[item]);
```

Comment:
There seems to be a lot of console logs. I am not sure if these are still necessary, but you may want to remove some of the less important ones just to keep it clean; or change them to give error messages should something go wrong.